src/Widgets/Card.jsx

```diff
@@ -17,7 +17,7 @@ const Card = ({details, images, cardClick, addOutfit}) => {
     )
   } else {
     return (
-      <div className="card">
+      <div className="card" onClick={cardClick.bind(this, details.id)}>
```

Comment:
Would love to chat about this binding as I think it could be useful in my component as well.